apps/src/lib/s2d.ts

```diff
@@ -241,3 +241,30 @@ export function getForecastTypeName(forecastType: ForecastType): string {
 
 	return nameMap[forecastType] ?? (forecastType as string);
 }
+
+/**
+ * Convert an S2D climate variable's id to the one to use in S2D API requests.
+ *
+ * @param variableId - The variable id used in the app, e.g. `"s2d_air_temp"`.
+ * @returns The variable id to use for S2D API requests, e.g. `"air_temp"`.
+ */
+export function getApiVariableId(variableId: string): string {
+	// For now, the only difference between the app's variable ids and the ones
+	// used in the API is simply a prefix.
+	return variableId.replace(/^s2d_/, '');
+}
+
+/**
+ * Convert an S2D frequency type to the one to use in S2D API requests.
+ *
+ * @param frequency - The frequency, e.g. `FrequencyType.SEASONAL`.
+ * @returns The frequency name to use for S2D API requests, e.g. `"seasonal"`
+ */
+export function getApiFrequencyName(frequency: S2DFrequencyType | string): string {
```

Comment:
Since it's an input to output, let's prefix `normalizeFor`

```suggestion
export function normalizeForApiFrequencyName(frequency: S2DFrequencyType | string): string {
```